src/mimic_joint_plugin.cpp

```diff
@@ -22,21 +22,23 @@ OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISE
 
 #include <roboticsgroup_gazebo_plugins/mimic_joint_plugin.h>
 
+#if GAZEBO_MAJOR_VERSION >= 8
```

Comment:
I would prefer something like (there's no actual need to use the whole namespaces):

```cpp
#if GAZEBO_MAJOR_VERSION >= 8
namespace math = ignition::math;
#else
namespace math = gazebo::math;
#endif
```

Then, you can use `math::clamp` in the code..

Reply:
Done!

Reply:
Great! Thanks! I will test it in my machine and then accept it! Thanks again!